### ISSUE TITLE
Allow for dynamically-generated certificates, keys, etc.

### ISSUE THREAD

**Issue by AimForNaN** [uNetworking/uWebSockets.js#1006]
Packages like `node-forge` let us dynamically generate self-signed certificates, private/public key pairs, etc. Developer experience can be significantly improved if we didn't have to save them to file and just pass them in directly.

**Comment by uNetworkingAB** [comment]
Yes a few have reported this, maybe it could be time to look at it. Very low prio.
